scripts/example_service

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python3
+
+# Copyright 2018 PickNik Robotics LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Author: Michael Lautman
+
+import math
+
+import rclpy
+
+from rclpy.node import Node
+
+from rqt_service_caller.srv import Sum
+
+
+class SumServer(Node):
+
+    def __init__(self):
+        super().__init__('compute_sum_server')
+        self.srv = self.create_service(Sum, 'compute_sum', self.sum_callback)
+
+    def sum_callback(self, request, response):
+        self.get_logger().info('incoming request:')
+        response.total = 0.0
+        for ix, val in enumerate(request.val.vals):
+            for jx, float_ in enumerate(val.floats):
+                self.get_logger().info('\trequest.vals[{}].floats[{}] = {}'.format(ix, jx, float_))
+                response.total += float_
+
+        return response
+
+
+def main(args=None):
+    rclpy.init(args=args)
+
+    node = SumServer()
+
+    rclpy.spin(node)
+
+    # Destroy the node explicitly
+    # (optional - Done automatically when node is garbage collected)
+    node.destroy_node()
+    rclpy.shutdown()
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
I don't think this package needs an example service script. You can easily test this with any existing demo node.

Reply:
The existing demo nodes are very basic and do not test corner cases such as setting arrays, setting fields with full message definitions etc.. I'll split the example node into a separate repo

Reply:
Why can't you use one of the existing demo nodes but need your own custom one?